docs/api/cli/mint.md

```diff
@@ -194,6 +199,11 @@ Mint an agent.
     required=True,
     help="Threshold for the minimum numbers required to run the service",
 )
+@click.option(
```

Comment:
See above; should be inferred from network selection. Not something user needs to bother with